runtime/bcutil/cfreader.c

```diff
@@ -2320,13 +2299,7 @@ checkAttributes(J9PortLibrary* portLib, J9CfrClassFile* classfile, J9CfrAttribut
 						goto _errorFound;
 					}
 					/* Capture the error if the outer_class_info_index points to an array class or other referencetype descriptor */
-					if (
-#if defined(J9VM_OPT_VALHALLA_FLATTENABLE_VALUE_TYPES)
-						bcvIsReferenceTypeDescriptor(classInfoUtf8)
-#else /* defined(J9VM_OPT_VALHALLA_FLATTENABLE_VALUE_TYPES) */
-						('[' == classInfoUtf8->bytes[0])
-#endif /* defined(J9VM_OPT_VALHALLA_FLATTENABLE_VALUE_TYPES) */
-					) {
+					if ('[' == classInfoUtf8->bytes[0]) {
```

Comment:
The message of `J9NLS_CFR_ERR_OUTER_CLASS_REFERENCETYPE_DESCRIPTOR` is
"Outer class is a ReferenceType descriptor in an InnerClasses attribute". 
It needs to be updated.

Reply:
Thanks. I updated the message to what it was before https://github.com/eclipse-openj9/openj9/pull/15053